doc/releases/release-notes-3.2.rst

```diff
@@ -529,6 +529,18 @@ Libraries / Subsystems
 
 * C Library
 
+  * Added Picolibc as a Zephyr module. Picolibc module is a footprint-optimized
+    full C standard library implementation that is configurable at the build
+    time.
+  * C library heap initialization call has been moved from the ``APPLICATION``
+    phase to the ``POST_KERNEL`` phase to allow calling the libc dynamic memory
+    management functions (e.g. ``malloc()``) during the application
+    initialization phase.
+  * Added ``strerror()`` and ``strerror_r()`` functions to the minimal libc.
```

Comment:
Does it make sense to add `:c:func` for these?

Reply:
No, https://github.com/zephyrproject-rtos/zephyr/pull/50462#discussion_r976258605

Reply:
Cool, thanks.